docs/aspnetcore-mvc.md

```diff
@@ -101,7 +101,32 @@ public IActionResult SignInWithIdp(string idp)
 }
 ```
 
-# Handling failures
+## Accessing OIDC Tokens
+
+To access OIDC tokens, AspNet Core provides the HttpContext.GetTokenAsync(...) extension method.  The following is an example of how to access OIDC tokens from your HomeController:
```

Comment:
I think is a good idea is we include a link for the extension method. 
Also, should we use markdown for `HttpContext.GetTokenAsync(...)` and `HomeController`?